### DISCUSSION TITLE
math menu

### DISCUSSION THREAD

**Discussion by SunnyCheung-cp** [Milkdown/milkdown#1050] [Q&A]
I'm implementing a math button
when I input y=x^2 and click the math button, it successfully displays
can we do it?

this is my test case: https://stackblitz.com/edit/github-z5jjcg?file=components%2FEditor.tsx,components%2Fmath.js

**Comment by Saul-Mirone** [comment]
The command you provided need to get the text in current selection and transform them into inline math node.

**Comment by SunnyCheung-cp** [comment]
@Saul-Mirone  Thank you for your help
I'm using the "mathInlineSchema" of "@milkdown/plugin-math", and converting the text to inline latex via "wrapIn", but it doesn't seem to respond
can you tell me how to do it in more detail, or do you have something similar for reference?

**Reply by Saul-Mirone** [reply]
WrapIn won't work for in this case, you should use something like [`NodeType.create`](https://prosemirror.net/docs/ref/#model.NodeType.create).
